test/integration/build_to_git_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright The Shipwright Contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+package integration_test
```

Comment:
```suggestion
// SPDX-License-Identifier: Apache-2.0

package integration_test
```

Reply:
Done!

Reply:
is this the case? looks the same to me. Seems MD suggestion didnt took place.

Reply:
Done!